engine/src/flutter/shell/platform/android/test/io/flutter/embedding/android/FlutterActivityAndFragmentDelegateTest.java

```diff
@@ -1098,7 +1100,66 @@ public void itForwardsOnUserLeaveHintToFlutterEngine() {
   @SuppressWarnings("deprecation")
   // TRIM_MEMORY_COMPLETE, TRIM_MEMORY_MODERATE, TRIM_MEMORY_RUNNING_LOW,
   // TRIM_MEMORY_RUNNING_MODERATE, TRIM_MEMORY_RUNNING_CRITICAL
-  public void itNotifiesDartExecutorAndSendsMessageOverSystemChannelWhenToldToTrimMemory() {
+  @Config(minSdk = Build.API_LEVELS.API_26)
+  public void
```

Comment:
I am still seeing a mix of @RunWith(AndroidJUnit4.class) and @Config. 

Can you walk me through the reason for using them in the same file?